qiskit/providers/__init__.py

```diff
@@ -43,19 +43,19 @@
 Version Support Policy
 ----------------------
 
-To enable providers to have time to adjust to changes in this interface
```

Comment:
This looks like your editor enforces a linebreak after 100 characters -- I think it would be good to leave lines like this unchanged. Could you check that your editor is synced with the Terra requirements, you could check that in the `.pylintrc` for the lint settings and `pyproject.toml` for black 🙂

Reply:
I reverted the changes to the other lines, but here almost the whole file is wrapped around 80 (which is what I also do by default), whereas the default in the settings is 100.